assets/js/main.js

```diff
@@ -0,0 +1,308 @@
+var taskName = document.getElementById("taskName");
+var assignee = document.getElementById("assignee");
+
+var addTaskButton = document.getElementById("addTaskButton");
+var clearTaskButton = document.getElementById("clearTaskButton");
+
+var tasksData = document.getElementById("tasksData");
+var tasksDataDone = document.getElementById("tasksDataDone");
+
+var tasksArray = [];
```

Comment:
you dont need to use "Array" as part of the name. its already obvious that its a list of tasks.